in_toto/model.go

```diff
@@ -150,6 +226,51 @@ func (l *Layout) InspectAsInterfaceSlice() []interface{} {
 	return inspectionsI
 }
 
+func validateLayout(layout Layout) error {
+	if layout.Type != "layout" {
+		return fmt.Errorf("invalid Type value for layout: should be 'layout'")
+	}
+
+	if _, err := time.Parse(ISO8601DateSchema, layout.Expires); err != nil {
```

Comment:
Just to make sure,  will this check blow up if `layout.Expires` is a slightly different flavor of `ISO8601`, e.g. with milliseconds or utc offset?